PyriteCore/src/world/Shadows/Lightmap.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "display/texture.h"
+#include <variant>
+
+template<class... Ts>
+struct overloaded : Ts... { using Ts::operator()...; };
```

Comment:
that should probably not be here, isn't there already one for camera projections anyway?